src/vm/jitinterfacegen.cpp

```diff
@@ -246,9 +248,11 @@ void InitJITHelpers1()
 
                 ECall::DynamicallyAssignFCallImpl(GetEEFuncEntryPoint(AllocateStringFastMP), ECall::FastAllocateString);
```

Comment:
We should plan to do the same treatment for the other allocation helpers in this list - in follow up PRs.